src/Microsoft.Diagnostics.DebugServices.Implementation/ThreadService.cs

```diff
@@ -59,7 +59,7 @@ public ThreadService(IServiceProvider services)
                     contextType = typeof(ArmContext);
                     break;
 
-                case (Architecture)6 /* Architecture.LoongArch64 */:
+                case Architecture.LoongArch64:
```

Comment:
.NET 6 will be out of support in five weeks, should we upgrade to net8.0 already?

Reply:
I'm not sure yet. I think we may want our tools to work with 6.0 for awhile longer than the out of support date.

Reply:
I wasn't talking the runtime versions SOS targets but the version required to run/host SOS.  You are right that it will target  old versions back to 3.1 or so.  When 6.0 goes out of service in November we can do make the Architecture.LoongArch64.